pkg/image/context.go

```diff
@@ -18,4 +18,23 @@ type Context struct {
 	ArtefactsDir string
 	// ImageDefinition contains the image definition properties.
 	ImageDefinition *Definition
+	// ArtifactSources contains the information necessary for the deployment of external artifacts.
+	ArtifactSources *ArtifactSources
+}
+
+type ArtifactSources struct {
```

Comment:
Not convinced this is the best package to have the definition at but it felt weird to keep `env` for it. Thoughts?

Reply:
I don't hate it. The image context describes everything needed to build a particular image. In this case, the sources are inputs for that, in the same way the image configuration directory is.